components/o3-button/stories/core/professional/button.stories.tsx

```diff
@@ -15,6 +15,15 @@ export default {
 			</div>
 		),
 	],
+	argTypes: Object.assign(ButtonStories.Button.argTypes, {
+		theme: {
+			options: ['default', 'inverse'],
+			mapping: {
+				default: '',
```

Comment:
To align with the terminology we are using in Figma?
```suggestion
			options: ['standard', 'inverse'],
			mapping: {
				standard: '',
```